registry.json

```diff
@@ -4043,5 +4043,19 @@
    "telegram": "HappyRaspberry",
    "operatorName": "Patrik",
    "github": "kolesarp1"
+  },
+  {
+  "ticker": "SOCK",
+   "poolId": "564e6152a892f02d3456bf3a5e00ad0aa3806369d1b152a8abae435b",
+   "social": {
+     "website": "https://sockstake.io",
+     "twitter": "ADAPoolSock",
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
+   },
+   "telegram": "SockStakeAdaPool",
```

Comment:
Please change this to be the handle of an individual and not a group or channel

Reply:
Updated, blank now.

Reply:
Please can you change this to be the handle of an individual and not a group or channel? @sockstakeadapool is a channel...